setup.py

```diff
@@ -105,9 +86,10 @@ def configuration(parent_package='', top_path=None):
         ],
 
         configuration=configuration,
-        setup_requires=SETUP_REQUIRES,
         install_requires=INSTALL_REQUIRES,
-        requires=REQUIRES,
```

Comment:
This is used to populate the PyPI entry, which does not pick up `install_requires`.